cypress/integration/chi-custom-elements/carousel.spec.js

```diff
@@ -42,7 +42,6 @@ describe('Carousel', () => {
         .find('button')
         .as('nextButton')
         .click()
-        .click()
```

Comment:
Since this scenario now has 8 elements, I have used a loop to click through all the elements one by one to finally reach the last element where the next button will be disabled.